feature-libs/product-configurator/rulebased/components/attribute/types/numeric-input-field/configurator-attribute-numeric-input-field.component.service.ts

```diff
@@ -48,6 +56,139 @@ export class ConfiguratorAttributeNumericInputFieldService {
       splitParts[1].length > numberDecimalPlaces
     );
   }
+
+  formatIntervalValue(
+    intervalValue: number,
+    decimalPlaces: number | undefined,
+    locale: string
+  ): string {
+    let formatted = formatNumber(
+      intervalValue,
+      locale,
+      '1.' + decimalPlaces + '-' + decimalPlaces
```

Comment:
This does not make sense in case decimal places is undefined. In that case, you rather want this attribute be composed with '0', correct?

Reply:
Check if decimalPlaces is set, if not it will be set to 0